container/libcontainer/handler_test.go

```diff
@@ -217,58 +217,34 @@ func TestParseLimitsFile(t *testing.T) {
 	}
 }
 
-func TestReferencedBytesStat(t *testing.T) {
+func TestGetReferencedKBytes(t *testing.T) {
 	//overwrite package variables
 	smapsFilePathPattern = "testdata/smaps%d"
-	clearRefsFilePathPattern = "testdata/clear_refs%d"
+	smaps_rollupFilePattern = "testdata/overwrite%d"
```

Comment:
Why have you removed setting of clear_refs file for tests?

Reply:
After the changes getReferencedKBytes no longer refers to clear_refs, so there is no point in overwriting it

Reply:
If clear_refs is no longer needed then please remove it also from other parts of code.

Reply:
My bad, I misslead with the explanation.  For this testcase cear_refs are not required. However I want to test behavior of `getReferencedBytes` when rollup is available and normal smaps not. That is why the variables look like this:
```
	smapsFilePathPattern = "testdata/smaps%d"
	smaps_rollupFilePattern = "testdata/overwrite%d"
```
so I can assure that one points to a file and other one does not

Reply:
Have you added a test when "testdata/overwrite%d" does not exist?